api/circulation.py

```diff
@@ -753,17 +767,26 @@ def fulfill(self, patron, pin, licensepool, delivery_mechanism, sync_on_failure=
             )
 
         if licensepool.open_access:
+            # We ignore the vendor-specific arguments when doing
+            # open-access fulfillment, because we just don't support
+            # partial fulfillment of open-access content.
```

Comment:
would this need to change for open-access audiobooks?

Reply:
or those let you download parts directly once you have the manifest, so it doesn't matter?

Reply:
Exactly. The manifest is publicly available and there's no need to route partial fulfillment requests through the circ manager.